sbol_utilities/sbol3_genbank_conversion.py

```diff
@@ -67,8 +70,104 @@ class GenBank_SBOL3_Converter:
     BIO_STRAND_REVERSE = -1
     DEFAULT_GB_REC_VERSION = 1
 
+    def __init__(self) -> None:
+        def build_component_genbank_extension(*, identity, type_uri) -> GenBank_SBOL3_Converter.Component_GenBank_Extension:
+            """A builder function to be called by the SBOL3 parser
+            when it encounters a Component in an SBOL file.
+            :param identity: identity for new component class instance to have
+            :param type_uri: type_uri for new component class instance to have
+            """
+            # `types` is required and not known at build time.
+            # Supply a missing value to the constructor, then clear
+            # the missing value before returning the built object.
+            obj = self.Component_GenBank_Extension(identity=identity, types=[sbol3.PYSBOL3_MISSING], type_uri=type_uri)
+            # Remove the placeholder value
+            obj.clear_property(sbol3.SBOL_TYPE)
+            return obj
+        # set up logging
+        log_level = logging.INFO
```

Comment:
I don't think that you want to set up logging here. Is there some reason you think it should be here?

Reply:
Actually, previously it was being set up inside the `main()` function, and the only thing it did was create an instance of the `GenBank_SBOL3_Converter` class. I used this for debugging and had logging setup here.

If our `GenBank_SBOL3_Converter` class is being invoked somewhere else in the library, we would intend to use the logging configs already defined in that file right? So should I move the logging setup back to `main()`?

Reply:
I'd suggest moving both the logging and the main to a test, instead of making main run a test.

Reply:
Alright, removing them from the main file.